packages/synapse-types/src/Table/ColumnModel.ts

```diff
@@ -2,11 +2,18 @@ import { ColumnType } from './ColumnType'
 
 // https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/table/ColumnModel.html
 export type ColumnModel = {
-  id: string // The immutable ID issued to new columns
-  name: string // The display name of the column
-  defaultValue?: string // The default value for this column. Columns of type FILEHANDLEID and ENTITYID are not allowed to have default values.
+  /** The immutable ID issued to new columns */
+  id: string
+  /** The display name of the column */
+  name: string
+  /** The default value for this column. Columns of type FILEHANDLEID and ENTITYID are not allowed to have default values. */
+  defaultValue?: string
   columnType: ColumnType
-  maximumSize?: number // A parameter for columnTypes with a maximum size. For example, ColumnTypeEnum.strings have a default maximum size of 50 characters, but can be set to a maximumSize of 1 to 1000 characters.
-  enumValues?: string[] // Columns of type string can be constrained to an enumeration values set on this list. The maximum number of entries for an enum is 100
+  /** A parameter for columnTypes with a maximum size. For example, ColumnTypeEnum.strings have a default maximum size of 50 characters, but can be set to a maximumSize of 1 to 1000 characters. */
+  maximumSize?: number
+  /** Required if using a columnType with a "_LIST" suffix. Describes the maximum number of values that will appear in that list. Value range 1-100 inclusive. Default 100 */
+  maximumListLength?: number
```

Comment:
This is a difficult diff to make sense of, but did you add the maximumListLength field?

Reply:
Yes